rdkit/Chem/UnitTestInchi.py

```diff
@@ -154,8 +156,12 @@ def setUp(self):
                                      'pubchem-hard-set.sdf.gz'),'r')
         self.dataset['problematic'] = ForwardSDMolSupplier(inf,sanitize=False,removeHs=False)
         with open(os.path.join(RDConfig.RDCodeDir, 'Chem/test_data',
-                               'pubchem-hard-set.inchi'),'rb') as inF:
-            self.dataset_inchi['problematic'] = load(inF)
+                               'pubchem-hard-set.inchi'),'r') as intF:
```

Comment:
The usual Windows CR+LF nightmare. I added a bunch of those in several Python tests in the past, and they were already merged in trunk.